topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>    > * idlist : points to a text file containing the list of sequence record identifiers to be selected.
+>    > * attribute : Regular expression pattern matched against the attributes of the sequence record. the value of this attribute is of the form : key:regular_pattern. ex:'family_name:Asteraceae'
+>    > * hasattribute : Selects sequence records having an attribute whose key = KEY.
+>    > * predicat : Python boolean expression to be evaluated for each sequence record. The attribute keys defined for each sequence record can be used in the expression as variable names. An extra variable named ‘sequence’ refers to the sequence record itself. ex: mode!="joined"
+>    > * lmax : Keeps sequence records whose sequence length is equal or shorter than lmax. ex : 100
+>    > * lmin : Selects sequence records whose sequence length is equal or longer than lmin. ex : 100
+>    {: .comment}
+>
+{: .hands_on}
+
+
+> ### {% icon question %} Questions
+>
+> 1. How do you verify the operation is successfull?
+> 2. How many sequences are kept? Discarded?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. you can search in the input file content the presence of `mode=joined` and same on the output file (just clicking the eye to visualize the content of each file and typing CTRL+C for example to search `mode=joined` in the file, or using a regex Galaxy tool for example). You can also at least look at the size of the output file, if smaller than input file, this is a first good indication.
+> > 2. You can use a Galaxy tool like `Line/Word/Character count of a dataset` to count the number of lines of each dataset (input and output of obigrep) and divided by 4 (as in a FastQ file, each sequence is represented by a block of 4 lines). 45 276 sequences for input file. 44 717 for output file. Thus 559 sequences discarded.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Assign each sequence record to the corresponding sample/marker combination with **NGSfilter**
+
+> ### {% icon hands_on %} Hands-on: Assigns sequence records to the corresponding experiment/sample based on DNA tags and primers
+>
+> 1. {% tool [NGSfilter](toolshed.g2.bx.psu.edu/repos/iuc/obi_ngsfilter/obi_ngsfilter/1.2.13) %} with the following parameters:
+>    - *"Parameter file"*: `wolf_diet_ngsfilter`
+>    - *"Read from file"*: `obigrep output`
+>    - *"Number of errors allowed for matching primers"*: `2`
+>    - *"Output data type"*: `fastq`
+>
+>
+>    > ### {% icon tip %} Tip: Be sure the text file is in tabular datatype
+>    >
+>    > * If you are sure the format is compatible with a tabular datatype, as this is the case here ;), you can manually change it, clicking on the eye of the "wolf_diet_ngsfilter.txt" dataset, then selecting the "Datatypes" section then affecting manually tabular and saving the operation
+>    >
+>    {: .tip}
+>
+>
+>    > ### {% icon comment %} Comment
```

Comment:
I think that it will be nice to move the content of this comment block to the beginning of the section in order to understand why should we run this command before running it.